paasta_tools/instance/kubernetes.py

```diff
@@ -200,29 +200,25 @@ async def job_status(
     kstatus["app_id"] = app_id
     kstatus["pods"] = []
     kstatus["replicasets"] = []
-    if verbose > 0:
-        num_tail_lines = calculate_tail_lines(verbose)
+    num_tail_lines = calculate_tail_lines(verbose)
+    kstatus["pods"] = await asyncio.gather(
+        *[pod_info(pod, client, num_tail_lines) for pod in pod_list]
+    )
```

Comment:
I think we don't want to show pods at the non-verbose level

Reply:
Perhaps we can leave the App Created Timestamp in the verbose level?